packages/react-dom/src/client/ReactInputSelection.js

```diff
@@ -12,7 +12,25 @@ import * as ReactDOMSelection from './ReactDOMSelection';
 import {ELEMENT_NODE} from '../shared/HTMLNodeType';
 
 function isInDocument(node) {
-  return containsNode(document.documentElement, node);
+  return (
+    node &&
```

Comment:
The value that gets passed in comes from `fbjs/lib/getActiveElement`, which returns a nullable `HTMLElement` (https://github.com/facebook/fbjs/blob/master/packages/fbjs/src/core/dom/getActiveElement.js#L21). The only instance where the return value would be null is if the util was unable to find a `document` object, which I think would only happen in SSR. But because it is theoretically nullable, all the operations in this file first confirm that `node` (or `elem`, in `hasSelectionCapabilities`) is truthy.

Reply:
We don't use `fbjs/lib/getActiveElement` anymore, and have copied it in the repo. Let's tighten this up? Feel free to change `getActiveElement` as you see fit.

Reply:
I want to lean towards the first, but I keep reasoning out of it in favor of the second. Body _could_ be null, and it really it should _never_ happen. But I've been surprised too much before :).

With the second example, do you need a try/catch?

Also: do you anticipate any problems with code downstream working with a document body that isn't attached?

Reply:
@nhunzaker No try/catch needed for the second example, and the ReactInputSelection plugin won’t have any issues; the code is already setup to handle detached DOM elements for a case where the active element becomes detached between when it is first read and cached and after React finishes committing an update. The second option has grown on me; I suggested it thinking it was silly, but now feel like it’s pretty reasonable. If we go with that one, should we add a comment explaining that document.body can be null?

Reply:
> The second option has grown on me; I suggested it thinking it was silly, but now feel like it’s pretty reasonable. If we go with that one, should we add a comment explaining that document.body can be null?

Let's go with it 👍